packages/babel-plugin-transform-typeof-symbol/test/helper.spec.js

```diff
@@ -0,0 +1,52 @@
+import * as babel from "@babel/core";
+import resolvePath from "resolve";
+import fs from "fs";
+
+import transformTypeofSymbol from "..";
+
+const resolve = path =>
+  new Promise((resolve, reject) =>
+    resolvePath(path, (err, path) => (err ? reject(err) : resolve(path))),
+  );
+const readFile = path =>
+  new Promise((resolve, reject) =>
```

Comment:
Can be refactored using [`util.promisify`](https://nodejs.org/api/util.html#util_util_promisify_original)
Example:

    const readFile = path => util.promisify(fs.readFile)(path, "utf8")

Reply:
No, it's not supported by Node 6.
When we'll drop it, we'll directly use `fs.promises`.